benchmarkapps/Benchmarks/StartupUsingDispatcher.cs

```diff
@@ -35,7 +35,8 @@ public void ConfigureServices(IServiceCollection services)
                         values: new { },
                         order: 0,
                         metadata: EndpointMetadataCollection.Empty,
-                        displayName: "Plaintext"),
+                        displayName: "Plaintext",
+                        address: null),
```

Comment:
Future design note:

I think this might be too coupled. We might need the ability to pluggably 'scavenge' addresses from endpoints, rather than requiring the addresses to be known when we create them. 

We also might need multiple addresses per endpoint.

I don't think these are blockers right now.

Reply:
Filed: https://github.com/aspnet/Routing/issues/563